src/jdk.compiler/share/classes/com/sun/tools/javac/comp/Flow.java

```diff
@@ -1810,19 +1810,19 @@ void newVar(JCVariableDecl varDecl) {
          */
         void letInit(DiagnosticPosition pos, VarSymbol sym) {
             if (sym.adr >= firstadr && trackable(sym)) {
-                if ((sym.flags() & EFFECTIVELY_FINAL) != 0) {
+                if (sym.isFlagSet(VarSymbolFlags.EFFECTIVELY_FINAL)) {
                     if (!uninits.isMember(sym.adr)) {
                         //assignment targeting an effectively final variable
                         //makes the variable lose its status of effectively final
                         //if the variable is _not_ definitively unassigned
-                        sym.flags_field &= ~EFFECTIVELY_FINAL;
+                        sym.clearFlag(VarSymbolFlags.EFFECTIVELY_FINAL);
                     } else {
                         uninit(sym);
                     }
                 }
                 else if ((sym.flags() & FINAL) != 0) {
                     if ((sym.flags() & PARAMETER) != 0) {
-                        if ((sym.flags() & UNION) != 0) { //multi-catch parameter
+                        if (sym.isFlagSet(VarSymbolFlags.UNION)) { //multi-catch parameter
                             log.error(pos, Errors.MulticatchParameterMayNotBeAssigned(sym));
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.